index.js

```diff
@@ -1,5 +1,11 @@
 /**
  * @typedef {import('unist').Node} Node
+ * @typedef {import('unist').Parent} Parent
+ */
+
+/**
+ * @template {Node} [T = Node]
```

Comment:
```suggestion
 * @template {Node} [Tree=Node]
```

Full name to clarify what this type is: you can pass a single node, not an enumeration of nodes.